definitions/divorce/json/CaseEventToFields/CaseEventToFields-deemed-and-dispensed-nonprod.json

```diff
@@ -62,10 +60,12 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "makeServiceDecision3",
-    "PageLabel": "Reason for refusal - Make Service Decision",
+    "PageLabel": "Refusal order draft",
     "PageDisplayOrder": 3,
     "PageColumnNumber": 1,
     "PageShowCondition": "ServiceApplicationRefusalReason=\"*\"",
-    "ShowSummaryChangeOption": "No"
+    "ShowSummaryChangeOption": "No",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/service-refusal-draft",
+    "RetriesTimeoutURLMidEvent": "120,120"
```

Comment:
Midevent callback to generate draft document. Would be cleaned up after the event is submitted